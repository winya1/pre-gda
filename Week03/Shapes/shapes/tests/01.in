4
0 0 1 0 0 1
1 1 1 4 5 1
-1 -1 3 -1 1 4
1 2 3 5 5 4
