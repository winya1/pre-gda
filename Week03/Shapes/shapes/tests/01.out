Case #1: 0.50 3.41 1.00 4.00
Case #2: 6.00 12.00 12.00 14.00
Case #3: 10.00 14.77 20.00 18.00
Case #4: 4.00 10.31 12.00 14.00
